go/felix/rules/endpoints.go

```diff
@@ -168,9 +195,22 @@ func (r *ruleRenderer) WorkloadEndpointToIptablesChains(epID *proto.WorkloadEndp
 	return []*Chain{&toEndpointChain, &fromEndpointChain}
 }
 
-func (r *ruleRenderer) HostDispatchChains(map[proto.HostEndpointID]*proto.HostEndpoint) []*Chain {
-	panic("Not implemented")
-	return nil
+func (r *ruleRenderer) HostDispatchChains(endpoints map[proto.HostEndpointID]*proto.HostEndpoint) []*Chain {
+
+	// Extract endpoint names.
+	names := make([]string, 0, len(endpoints))
+	for _, endpoint := range endpoints {
+		names = append(names, endpoint.Name)
```

Comment:
This will only work if the endpoint you're passing in has its name defined (which isn't guaranteed for host endpoints), have you thought about that, design-wise?  Are you planning for the endpoints you pass in here to be "resolved", i.e. to have their names filled in?

Another alternative that I tossed around was to base our chain names on the ID instead of the name.